concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -76,10 +77,21 @@ def test_provider_forces_subtask_results_for_task_which_was_already_submitted_co
         assert response_1.status_code   == 202
 
         self._test_database_objects(
-            last_object_type         = message.concents.ForceSubtaskResults,
+            last_object_type         = message.concents.AckReportComputedTask,
             task_id                  = '2',
             receive_delivered_status = False,
         )
+        self._assert_stored_message_counter_increased(increased_by = 3)
+        self._test_subtask_state(
+            task_id                      = '2',
+            subtask_id                   = 'xxyyzz',
+            subtask_state                = 'FORCING_ACCEPTANCE',
```

Comment:
Please use values directly from SubtaskState enum, as they can change in the future. If `.name` or `.value` of the enum is needed, please use it inside `_test_subtask_state` function. This should be applied in every usage.

Reply:
:+1: